ssl/d1_lib.c

```diff
@@ -233,6 +234,8 @@ long dtls1_ctrl(SSL *s, int cmd, long larg, void *parg)
             return 0;
         s->d1->mtu = larg;
         return larg;
+    case DTLS_CTRL_GET_DATA_MTU:
+        return (long)dtls1_get_data_mtu(s);
```

Comment:
This is why I hate SSL_ctrl for things like this (at some point I'd like to convert lots of the existing SSL_ctrls to be functions). The cast to long is just wrong and horrible. I'm wondering whether a proper function rather than a ctrl would be better?